packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -350,10 +372,16 @@ class OverlayState extends State<Overlay> with TickerProviderStateMixin {
     final List<Widget> onstageChildren = <Widget>[];
     final List<Widget> offstageChildren = <Widget>[];
     bool onstage = true;
+    bool excludeSemantics = false;
     for (int i = _entries.length - 1; i >= 0; i -= 1) {
       final OverlayEntry entry = _entries[i];
       if (onstage) {
-        onstageChildren.add(new _OverlayEntry(entry));
+        onstageChildren.add(new ExcludeSemantics(
```

Comment:
This isn't going to work because part of the contract Overlay has with its entries is that they'll be contained in a Stack.